transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -353,6 +356,7 @@ final void epollRdHupReady() {
                 // Clear the EPOLLRDHUP flag to prevent continuously getting woken up on this event.
                 clearEpollRdHup();
             }
+
```

Comment:
I think the new line makes the white space separation more consistent...are you OK with leaving it?